src/components/Navbar.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const HeaderStyle = {
+  color: '#b6ad90',
+};
+
+const Navbar = () => (
+  <nav>
+    <h1 style={HeaderStyle}>Math Magician</h1>
+    <ul className="nav_links">
+      <Link to="/"><li>Home</li></Link>
+      <Link to="/Calculator"><li>Calculator</li></Link>
+      <Link to="/Quotes"><li>Quotes</li></Link>
+    </ul>
+  </nav>
```

Comment:
- [ ] Kindly try to follow the wireframes that are provided in the project requirements, the name of the project should be on the left of the header and the nav links should be on the right of the header.

![image](https://user-images.githubusercontent.com/77190203/160655970-9daf38a8-5ed2-4190-9f4a-16e36d2969a3.png)